src/components/WishList/WishListItem.jsx

```diff
@@ -0,0 +1,135 @@
+import { useAuth, useCart, useWishList } from "contexts";
+import { useToast, useUpdateCart } from "custom-hooks/";
+import { useState } from 'react';
+import ShoppingCartIcon from '@mui/icons-material/ShoppingCart';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import { deleteProductInWishList, postToCart } from "utils";
+
+
+const WishListItem = ({ wishListItem }) => {
+
+    const [isOngoingNetworkCall, setIsOngoingNetworkCall] = useState(false); 
```

Comment:
instead of separate state, do make use of the`loading` value from the whishlist state

Reply:
Yes, I shall make use of that state.